packages/plugin-core/src/commands/DeleteCommand.ts

```diff
@@ -73,6 +68,60 @@ export class DeleteCommand extends BasicCommand<CommandOpts, CommandOutput> {
     return nodePosition?.end.line;
   }
 
+  /**
+   * When Delete Command is ran from Tree view, it gets Noteprops as args
+   */
+  private isNotePropsArgs(opts: CommandOpts) {
```

Comment:
suggestion:  Maybe make a commandUtils.ts file or something under commands that could share this function between the two commands you have here?